stdlib/http/src/main/ballerina/src/http/websocket/websocket_client.bal

```diff
@@ -188,6 +192,22 @@ public type WebSocketClientConfiguration record {|
     ClientSecureSocket? secureSocket = ();
     int maxFrameSize = 0;
     boolean webSocketCompressionEnabled = true;
+    int timeoutInSecs = 300;
```

Comment:
I think that naming this as `timeoutInSecs` will cause confusion with the `idleTimeoutInSeconds` config. Shall we come up with a better name? Maybe something like `handShakeTimeoutInSeconds`?